types/jscodeshift/index.d.ts

```diff
@@ -1,9 +1,11 @@
 // Type definitions for jscodeshift 0.6
 // Project: https://github.com/facebook/jscodeshift#readme
 // Definitions by: Brie Bunge <https://github.com/brieb>
+//                 Brian Jacobel <https://github.com/bjacobel>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.9
 
 import core = require("./src/core");
+import testUtils = require('./src/testUtils');
```

Comment:
nit: `"` instead of `'` for consistency

Reply:
Does a `// <reference ... />` work here instead of a noop import?

Reply:
Also, why is this import necessary?

Reply:
It's needed to avoid an `Error: Unused file` from the parser in types-publisher. However, a reference works as well, and makes more sense, so I will change to that.

Thank you for the review!